dao/src/main/java/greencity/repository/NotificationParameterRepository.java

```diff
@@ -4,6 +4,12 @@
 import org.springframework.data.jpa.repository.JpaRepository;
 import org.springframework.stereotype.Repository;
 
+import java.util.List;
+
 @Repository
 public interface NotificationParameterRepository extends JpaRepository<NotificationParameter, Long> {
+    /**
+     * {@inheritDoc}
+     */
```

Comment:
Don't see any reason for override findAll, JpaRepository have default methods findAll();